usaspending_api/transactions/delta_models/detached_award_procurement.py

```diff
@@ -0,0 +1,323 @@
+    "type_of_contract_pric_desc": {"delta": "STRING", "postgres": "TEXT"},
+    "type_of_contract_pricing": {"delta": "STRING", "postgres": "TEXT"},
+    "type_of_idc": {"delta": "STRING", "postgres": "TEXT"},
+    "type_of_idc_description": {"delta": "STRING", "postgres": "TEXT"},
+    "type_set_aside": {"delta": "STRING", "postgres": "TEXT"},
+    "type_set_aside_description": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_legal_enti": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_uei": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_unique_ide": {"delta": "STRING", "postgres": "TEXT"},
+    "undefinitized_action": {"delta": "STRING", "postgres": "TEXT"},
+    "undefinitized_action_desc": {"delta": "STRING", "postgres": "TEXT"},
+    "unique_award_key": {"delta": "STRING", "postgres": "TEXT"},
+    "updated_at": {"delta": "TIMESTAMP", "postgres": "TIMESTAMP"},
+    "us_federal_government": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "us_government_entity": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "us_local_government": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "us_state_government": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "us_tribal_government": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "vendor_alternate_name": {"delta": "STRING", "postgres": "TEXT"},
+    "vendor_alternate_site_code": {"delta": "STRING", "postgres": "TEXT"},
+    "vendor_doing_as_business_n": {"delta": "STRING", "postgres": "TEXT"},
+    "vendor_enabled": {"delta": "STRING", "postgres": "TEXT"},
+    "vendor_fax_number": {"delta": "STRING", "postgres": "TEXT"},
+    "vendor_legal_org_name": {"delta": "STRING", "postgres": "TEXT"},
+    "vendor_location_disabled_f": {"delta": "STRING", "postgres": "TEXT"},
+    "vendor_phone_number": {"delta": "STRING", "postgres": "TEXT"},
+    "vendor_site_code": {"delta": "STRING", "postgres": "TEXT"},
+    "veteran_owned_business": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "veterinary_college": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "veterinary_hospital": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "woman_owned_business": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+    "women_owned_small_business": {"delta": "BOOLEAN", "postgres": "BOOLEAN"},
+}
+
+
+DETACHED_AWARD_PROCUREMENT_DELTA_COLUMNS = {k: v["delta"] for k, v in DETACHED_AWARD_PROCUREMENT_COLUMNS.items()}
+DETACHED_AWARD_PROCUREMENT_POSTGRES_COLUMNS = {k: v["postgres"] for k, v in DETACHED_AWARD_PROCUREMENT_COLUMNS.items()}
+
+detached_award_procurement_create_sql_string = fr"""
+    CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
+        {", ".join([f'{key} {val}' for key, val in DETACHED_AWARD_PROCUREMENT_DELTA_COLUMNS.items()])}
+    )
+    USING DELTA
+    LOCATION 's3a://{{SPARK_S3_BUCKET}}/{{DELTA_LAKE_S3_PATH}}/{{DESTINATION_DATABASE}}/{{DESTINATION_TABLE}}'
+"""
+
+detached_award_procurement_load_sql_string = fr"""
+
+"""
```

Comment:
I would say leave out this variable because it will not be used.

Reply:
Removed